doc/releases/changelog-0.23.0.md

```diff
@@ -140,59 +195,118 @@
   `method='pyscf'` in :func:`~.molecular_hamiltonian`.
 
   ```python
-  import pennylane as qml
-  from pennylane import numpy as np
-  
   symbols = ["H", "H"]
   geometry = np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 2.0]])
   hamiltonian, qubits = qml.qchem.molecular_hamiltonian(symbols, geometry, method='pyscf')
   ```
 
+  Summary of QChem changes:
   - New functions are added for computing multipole moment molecular integrals
-    [(#2166)](https://github.com/PennyLaneAI/pennylane/pull/2166)
   - New functions are added for building a differentiable dipole moment observable
-    [(#2173)](https://github.com/PennyLaneAI/pennylane/pull/2173)
   - External dependencies are replaced with local functions for spin and particle number observables
-    [(#2197)](https://github.com/PennyLaneAI/pennylane/pull/2197)
-    [(#2362)](https://github.com/PennyLaneAI/pennylane/pull/2362)
   - New functions are added for building fermionic and qubit observables
-    [(#2230)](https://github.com/PennyLaneAI/pennylane/pull/2230)
   - A new module is created for hosting openfermion to pennylane observable conversion functions
-    [(#2199)](https://github.com/PennyLaneAI/pennylane/pull/2199)
-    [(#2371)](https://github.com/PennyLaneAI/pennylane/pull/2371)
   - Expressive names are used for the Hartree-Fock solver functions
-    [(#2272)](https://github.com/PennyLaneAI/pennylane/pull/2272)
   - These new additions are added to a feature branch
-    [(#2164)](https://github.com/PennyLaneAI/pennylane/pull/2164)
   - The efficiency of computing molecular integrals and Hamiltonian is improved
-    [(#2316)](https://github.com/PennyLaneAI/pennylane/pull/2316)
   - The qchem and new hf modules are merged
-    [(#2385)](https://github.com/PennyLaneAI/pennylane/pull/2385)
   - The 6-31G basis set is added to the qchem basis set repo
-    [(#2372)](https://github.com/PennyLaneAI/pennylane/pull/2372)
   - The dependency on openbabel is removed
-    [(#2415)](https://github.com/PennyLaneAI/pennylane/pull/2415)
   - The tapering functions are added to qchem
-    [(#2426)](https://github.com/PennyLaneAI/pennylane/pull/2426)
   - Differentiable and non-differentiable backends can be selected for building a Hamiltonian
-    [(#2441)](https://github.com/PennyLaneAI/pennylane/pull/2441)
   - The quantum chemistry functionalities are unified
-    [(#2420)](https://github.com/PennyLaneAI/pennylane/pull/2420)
-    [(#2465)](https://github.com/PennyLaneAI/pennylane/pull/2465)
-    [(#2454)](https://github.com/PennyLaneAI/pennylane/pull/2454)
 
-* Adds a MERA template.
+<h4>Pattern matching optimization 🔎 💎 </h4>
```

Comment:
@antalszava your call here, but you could merge the following subsections:

- New transforms for compilation and transpilation

  * Pattern matching
  * Swap-based transpiler

- A growing collection of templates

  * HilbertSchmidt and the LocalHilbertSchmidt 
  * qml.MERA

Reply:
That's a fair point, I was considering that. Then it came to mind that in this release we could afford to have sections be dedicated to a specific transform/template, so thought that the way they convey more information to the users. Having said that, happy to change as suggested, if we'd like that.

Reply:
@antalszava completely up to you :) I don't mind either approach - feel free to leave it if you prefer